cosmos/scheduler/cmd/scheduler.go

```diff
@@ -17,12 +19,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
+const (
+	nfsLocation = "location-nfs-mount-v1"
+	scalityRing = "location-ring-s3-v1"
```

Comment:
Was temporarily using `scalityRing` as an explicit check to return early but no longer needed. This can be deleted.